src/Model/Contact.php

```diff
@@ -1896,6 +1896,14 @@ public static function updateAvatar($avatar, $uid, $cid, $force = false)
 			$data = [$contact["photo"], $contact["thumb"], $contact["micro"]];
 		}
 
+		foreach ($data as $image_uri) {
+			$image_rid = Photo::ridFromURI($image_uri);
+			if ($image_rid && !Photo::exists(['resource-id' => $image_rid, 'uid' => $uid])) {
+				Logger::info('Regenerating avatar for contact uid ' . $uid . ' cid ' . $cid . ' missing photo ' . $image_rid . ' avatar ' . $contact['avatar']);
```

Comment:
Plz use the second argument of `Logger::info($text, array $args);` to add the data, you currently concatinated. The array should be defined like `['uid' => $uid, 'cid' => $cid, ...]` and the text would be `Regenarting avatar.`

Reply:
done

Reply:
What's the convention here, do I click "Resolve conversation", or should the original requester do that?

Reply:
Doesn't matter :) either you say you're done or the requested says "alright". Normally we're checking resolved conversations as well